zeppelin-zengine/src/main/java/org/apache/zeppelin/helium/Helium.java

```diff
@@ -411,7 +412,7 @@ public HeliumPackageSuggestion suggestApp(Paragraph paragraph) {
    * @return ordered list of enabled buildBundle package
    */
   public List<HeliumPackage> getBundlePackagesToBundle() {
-    Map<String, List<HeliumPackageSearchResult>> allPackages = getAllPackageInfoWithoutRefresh();
+    Map<String, List<HeliumPackageSearchResult>> allPackages = getAllPackageInfo();
```

Comment:
Hmmm, not sure... I don't think that would be a good approach as, if I understand your comment correctly, we would then have REST API calls inside REST API calls. We have 4 replacements of withoutRefresh to withRefresh: getAllEnabledPackages, getEnabledPackageInfo(pkgName), saveConfig() and suggest(). I think it's fine, for these 4 methods, to get up-to-date package info. Not having so, leads to inconsistencies. And the performance hit is not that bad. Do you agree?